.github/CHANGELOG.md

```diff
@@ -128,30 +341,35 @@
   Total shots:  300
   ```
 
-* VQE problems can now intuitively been set up by passing the Hamiltonian 
-  as an observable. [(#1474)](https://github.com/PennyLaneAI/pennylane/pull/1474)
+<h4>Containerization support</h4>
 
-  ``` python
-  dev = qml.device("default.qubit", wires=2)
-  H = qml.Hamiltonian([1., 2., 3.],  [qml.PauliZ(0), qml.PauliY(0), qml.PauliZ(1)])
-  w = qml.init.strong_ent_layers_uniform(1, 2, seed=1967)
-  
-  @qml.qnode(dev)
-  def circuit(w):
-      qml.templates.StronglyEntanglingLayers(w, wires=range(2))
-      return qml.expval(H)
+* Docker support for building PennyLane with support for all interfaces (TensorFlow,
+  Torch, and Jax), as well as device plugins and QChem, for GPUs and CPUs, has been added.
+  [(#1372)](https://github.com/PennyLaneAI/pennylane/issues/1372)
+
+  The build process using Docker and Makefile works as follows:
+
+  Building a core PennyLane image:
```

Comment:
Seems like these should be bullet points?

Reply:
Good catch! Updated